tests/src/test/scala/whisk/core/controller/test/ControllerTestCommon.scala

```diff
@@ -89,6 +89,8 @@ protected trait ControllerTestCommon
   val logStore = SpiLoader.get[LogStoreProvider].instance(actorSystem)
   val activationStore = SpiLoader.get[ActivationStoreProvider].instance(actorSystem, materializer, logging)
 
+  val ActivationIdHeaderInLowercase = ActivationIdHeader.toLowerCase
+
```

Comment:
I think it would be better to keep it because I restored the previous `exist` logic.
Unlike a normal action, we are unable to get the value of `ActivationID` from the response as so unable to use `contain` logic that Markus suggested.

```scala
headers should contain(RawHeader(ActivationIdHeader, _)) // does not work
```

Reply:
I'd propose to make the global setting lowercased then, and use that throughout. WDYT?

Reply:
Sorry, what does global setting mean?

Reply:
The value in ‘CustomHeaders’?